index.html

```diff
@@ -351,6 +351,61 @@ <h4 class="featureText">Terminal</h4>
           </li>
         </ul>
       </section>
+      <section>
+        <ul class="flex-container" id="formContainer">
+          <li>
+            <p id="contactDescription">I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</p>
+          </li>
+          <li>
+            <form action="https://formspree.io/f/xnqyndrw" method="POST">
+              <ul class="flex-container" id="formElements">
+                <li class="formElement">
+                  <input class="fElement" type="text" name="name" placeholder="Full name" maxlength="30" required>
+                </li>
+                <li class="formElement">
+                  <input type="email" name="email" placeholder="Email address" required>
+                </li>
+                <li class="formElement">
+                  <textarea name="message" placeholder="Enter text here" maxlength="500" required></textarea>
+                </li>
+                <li id="submitForm">
+                  <button class="button" id="subBut" type="submit">Get in touch</button>
+                </li>
+              </ul>
+            </form>
+          </li>
+        </ul>
```

Comment:
- [ ] Great job working on this section! 🚀 However, I think that using a `<ul>` tag here is not semantically correct as these elements don't represent a list. I suggest you remove the list and keep the section as your flex container for the paragraph and the form! 🤓 🔧